Kernel/KerrGeoOrbit.m

```diff
@@ -925,20 +941,24 @@
 	If[e>0,
 		\[CapitalDelta]tr=TimeOfMinoFastSpecR[a,p,e,x,{\[CapitalUpsilon]r,\[Lambda]rSample}];
 		\[CapitalDelta]\[Phi]r=PhiOfMinoFastSpecR[a,p,e,x,{\[CapitalUpsilon]r,\[Lambda]rSample}],
-		\[CapitalDelta]tr[\[Lambda]_?NumericQ]:=0;
-		\[CapitalDelta]tr[\[Lambda]_List]:=\[CapitalDelta]tr[#]&/@\[Lambda];
-		\[CapitalDelta]\[Phi]r[\[Lambda]_?NumericQ]:=0;
-		\[CapitalDelta]\[Phi]r[\[Lambda]_List]:=\[CapitalDelta]\[Phi]r[#]&/@\[Lambda];
+		\[CapitalDelta]tr[\[Lambda]_(*?NumericQ*)]:=0;
+		(*\[CapitalDelta]tr[\[Lambda]_List]:=\[CapitalDelta]tr[#]&/@\[Lambda];*)
+		SetAttributes[\[CapitalDelta]tr,Listable];
+		\[CapitalDelta]\[Phi]r[\[Lambda]_(*?NumericQ*)]:=0;
+		(*\[CapitalDelta]\[Phi]r[\[Lambda]_List]:=\[CapitalDelta]\[Phi]r[#]&/@\[Lambda];*)
+		SetAttributes[\[CapitalDelta]\[Phi]r,Listable]
```

Comment:
I would suggest just removing, rather than commenting out code here (and elsewhere).

Reply:
Before doing that, I wanted to ask one of the Chapel Hill guys what the purpose of restricting these definitions to numerical values only was. There might be something I am missing.

Reply:
For the spectral integration/Fourier sums I restricted the functions to numerical values because the series output could be unwieldy if you gave it a variable as input. I figured if people wanted analytic forms they would not be using the spectral method. I do not think that removing this restriction should break anything, but I would need to double check. If you all think it would be good to remove this restriction I can go in and make that change.

Reply:
Thanks @znasipak . Related question, is the ability to thread these functions over lists used anywhere internally? If not, we can drop setting the Listable attribute for these intermediate results. (The pure functions outputted at the end for the trajectory are already set to be listable.)

Reply:
@MvdMeent This is a relic of past versions of the code where the Listable attribute was not set for the functions that were returned by KerrGeoOrbit. If the pure functions provide this attribute then I think you are right and we can drop this setting for the intermediate result.